models.js

```diff
@@ -29,7 +31,9 @@ const masterSwram = [
     false, false, false, false];
 const tube65c02 = new Model("Tube65C02", [], ["tube/6502Tube.rom"], false, false);
 export const allModels = [
-    new Model("BBC B", ["B"], ["os.rom", "BASIC.ROM", "b/DFS-0.9.rom"], true, false, beebSwram, I8271),
+    new Model("BBC B", ["B"], ["os.rom", "BASIC.ROM", "b/DFS-0.9.rom"], true, false, beebSwram, I8271, null, false, false),
+    new Model("BBC B with Teletext", ["BTeletext"], ["os.rom", "BASIC.ROM", "b/DFS-0.9.rom", "ATS-3.0.ROM"], true, false, beebSwram, I8271, null, true),
```

Comment:
Looks like these need to be case-matched with the ROM files you checked in :)

```suggestion
    new Model("BBC B with Teletext", ["BTeletext"], ["os.rom", "BASIC.ROM", "b/DFS-0.9.rom", "ats-3.0.ROM"], true, false, beebSwram, I8271, null, true),
```

Reply:
It's a functional issue: only windows web servers are case insensitive :) Mac and UNIX (and AWS and CloudFront and...) are all case-sensitive, so I was getting "404 Not found" for these. :)